### PR TITLE
using-jquery-core/utility-methods: add type checking section

### PR DESCRIPTION
**Pull request by arthurvr**
This PR adds info about type checking utility methods to `using-jquery-core/utility-methods`. Would fix https://github.com/jquery/learn.jquery.com/issues/404 on the while.

### CODE REVIEW

**Comment by arthurvr** [comment]
Somebody to merge this?

**Comment by arthurvr** [comment]
@RedWolves Updated my PR.

File: page/javascript-101/testing-type.md [hunk 1]
```diff
@@ -44,4 +44,4 @@ if ( Object.prototype.toString.call( myArray ) === "[object Array]" ) {
 }
 ```
 
-jQuery also offers utility methods to help determine the type of an arbitrary value.
+jQuery also offers [utility methods](../using-jquery-core/utility-methods.md#testing-type) to help determine the type of an arbitrary value.
```
Comment (RedWolves) [inline page/javascript-101/testing-type.md]: The link doesn't work it goes to /javascript-101/using-jquery-core/utility-methods.md#testing-type

You don't need the ../

You also don't need the .md

Comment (RedWolves) [inline page/javascript-101/testing-type.md]: you do need a leading /

this will work /using-jquery-core/utility-methods#testing-type

Comment (arthurvr) [inline page/javascript-101/testing-type.md]: @RedWolves fixed.

**Comment by arthurvr** [comment]
@RedWolves Updated PR.

**Comment by arthurvr** [comment]
@RedWolves Any more review on this?

**Comment by RedWolves** [comment]
nope looks good.  Want to go ahead and land this? You have access now it looks like.

**Pull request closed without merging**